TWC 2019 Group Stage recap article (#2196)
ppy/osu-wiki: news/2019-04-24-TWC-2019-Group-Stage-Recap.md

---
layout: post
title: "osu!taiko World Cup - Group Stage Recap"
date: 2019-04-24 14:00:00 +0000
---

*sniff sniff...* do you smell that? Yeah, smells like ...*sniff*... poppin' gamers in here. What's poppin' gamers? The osu!taiko World Cup is in the limelight once again, with expectations higher than ever before. Read on to find out what happened in the Group Stage last weekend!

![](/wiki/shared/news/banners/TWC_2019_banner.jpg)

> ### Navigation

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> - [Mappool](#mappool)

Reviewer: ```suggestion
```

> - [Tournament Matches](#matches)

Reviewer: ```suggestion
```

> - [Looking Ahead](#ahead)

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

The sound of thundering drums marks the return of the osu!taiko World Cup, ready to march onward for its 9th installment. We've come a long way since our inception in 2011—long gone are the days where multiple country teams run rampant, with dominating performances from the likes of both Japan Team A and B, and curious oddities such as the great Italy-Netherlands Alliance of 2011-2013.

> This year is special in the sense that it is the first time the osu!taiko World Cup has had 32 unique countries pariticpating in a battle for the very top. With the addition of 8 teams from last year and an upgrade from best-of-7 to best-of-9 matches, last weekend's Group Stage was almost thrice as grand (and busy) as any before it—a momentous occasion and a cause for celebration for all osu!taiko fans and supporters alike.

Reviewer: ```suggestion
This year is special in the sense that it is the first time the osu!taiko World Cup has had 32 unique countries participating in a battle for the very top. With the addition of 8 teams from last year and an upgrade from best-of-7 to best-of-9 matches, last weekend's Group Stage was almost thrice as grand (and busy) as any before it—a momentous occasion and a cause for celebration for all osu!taiko fans and supporters alike.
```

Without further ado, let's get right into the Group Stage!

> 

Reviewer: ```suggestion

### Navigation

- [Mappool](#mappool)
- [Tournament Matches](#matches)
- [Looking Ahead](#ahead)

```

> ## <a name="mappool" id="mappool"></a>Mappool

Reviewer: ```suggestion
## <a name="mappool" id="mappool"></a>Mappool

```

**Conversely, [Shano x 40mP - Sanpo Michi](https://osu.ppy.sh/beatmapsets/62991#taiko/186452) holds the title of most banned map of the week, with 18 bans to 17 picks.** Understandably, it was clear that many teams didn’t want to risk tackling this devilishly tricky 120 BPM map on Hidden. Finally, [CyoucyoP feat.Hatsune Miku - End of the World](https://osu.ppy.sh/beatmapsets/215476#taiko/506018) received the least attention at 16 interactions (10 picks and 6 bans). It's a bit disappointing, but the map has made more than its fair share of appearances in past TWCs; it's certainly not the end of the world as the song would otherwise suggest.

## <a name="matches" id="matches"></a>Tournament Matches

> 

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

### Saturday

> 

Reviewer: ```suggestion
```

### Sunday

> --------

Reviewer: ```suggestion
```

## <a name="ahead" id="ahead"></a>Looking Ahead: Round of 16

**Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double-elimination tournament, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages.

**The first match of the RO16 Stage will take place Saturday, 27 April at 11:00 UTC.** There we will see a much-anticipated match between two veteran countries: Japan and France. Be sure to catch the stream on the [osu!live Twitch channel](https://www.twitch.tv/osulive)!

**Finally, here's a reminder to buy your banners today!** In addition to expressing your support for a chosen team, your contribution will also go towards funding the prize pool for this tournament. You can purchase the banners [here](https://osu.ppy.sh/store/products/ABC).

> --------

Reviewer: ```suggestion
--------

```

---
layout: post
title: "osu!taiko World Cup - Group Stage Recap"
date: 2019-04-24 14:00:00 +0000
---

*sniff sniff…* do you smell that? Yeah, smells like ...*sniff*... poppin' gamers in here. What's poppin' gamers? The osu!taiko World Cup is in the limelight once again, with expectations higher than ever before. Read on to find out what happened in the Group Stage last weekend!

![](/wiki/shared/news/banners/TWC_2019_banner.jpg)

> ### Navigation

Reviewer: Ignore this, my review will fix it.

**Conversely, [Shano x 40mP - Sanpo Michi](https://osu.ppy.sh/beatmapsets/62991#taiko/186452) holds the title of most banned map of the week, with 18 bans to 17 picks.** Understandably, it was clear that many teams didn’t want to risk tackling the devilishly tricky 120BPM map on Hidden. Finally, [CyoucyoP feat.Hatsune Miku - End of the World](https://osu.ppy.sh/beatmapsets/215476#taiko/506018) received the least attention at 16 interactions - 10 picks and 6 bans. A bit disappointing, but the map has made more than its fair share of appearances in past TWCs - it’s certainly not the end of the world as the song would otherwise suggest.

--------
## <a name="matches" id="matches"></a>Tournament Matches

> ### Saturday:

Reviewer 2: ```suggestion
### Saturday
```

> ### Sunday:

Reviewer 2: ```suggestion
### Sunday
```

--------

> If you missed out on any of the action last weekend, you can always watch the VODs on the [osu!live Twitch Channel](https://www.twitch.tv/osulive/videos).

Reviewer 2: ```suggestion
If you missed out on any of the action last weekend, you can always watch the VODs on the [osu!live Twitch channel](https://www.twitch.tv/osulive/videos).
```

> That concludes all the matches for now! Let's wave our goodbyes to the 16 teams that were narrowly defeated last weekend - we thank you for participating and hope to see you again next year. Good luck to all the remaining teams in this tournament!

Reviewer 2: ```suggestion
That concludes all the matches for now! Let's wave our goodbyes to the 16 teams that were narrowly defeated last weekend. We thank you for participating and hope to see you again next year. Good luck to all the remaining teams in this tournament!
```

--------
## <a name="ahead" id="ahead"></a>Looking Ahead: Round of 16

> **Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double elimination format, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages. 

Reviewer 2: ```suggestion
**Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double-elimination tournament, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages.
```

> **The first match of the RO16 Stage will take place Saturday 11:00 UTC, 27th of April.** There we will see a much anticipated match between two veteran countries - Japan vs. France. Be sure to catch the stream on the [osu!live twitch channel!](https://www.twitch.tv/osulive). 

Reviewer 2: ```suggestion
**The first match of the RO16 Stage will take place Saturday, 27 April at 11:00 UTC.** There we will see a much-anticipated match between two veteran countries: Japan and France. Be sure to catch the stream on the [osu!live Twitch channel](https://www.twitch.tv/osulive)!
```

> **Finally, a reminder to buy your banners today!** In addition to expressing your support for a chosen team, your contribution will also go towards funding the prize pool for this tournament. You can purchase the banners [here](https://osu.ppy.sh/store/products/ABC).

Reviewer 2: ```suggestion
**Finally, here's a reminder to buy your banners today!** In addition to expressing your support for a chosen team, your contribution will also go towards funding the prize pool for this tournament. You can purchase the banners [here](https://osu.ppy.sh/store/products/ABC).
```

--------
And that's all from me this week! I’d like to take this opportunity to thank [PuffBuck](https://osu.ppy.sh/users/4234525) for his help on collating information. I look forward to working closely with him in the coming weeks. If you have any feedback or suggestions about the format of these recaps, feel free to drop me a PM, or leave a comment down below. Otherwise, see you next week!

> Remember, rhythm is only a click away!

Reviewer 2: probably not needed, you already said goodbye

Reviewer: It's a reminder, not a goodbye.

**Conversely, [Shano x 40mP - Sanpo Michi](https://osu.ppy.sh/beatmapsets/62991#taiko/186452) holds the title of most banned map of the week, with 18 bans to 17 picks.** Understandably, it was clear that many teams didn’t want to risk tackling this devilishly tricky 120 BPM map on Hidden. Finally, [CyoucyoP feat.Hatsune Miku - End of the World](https://osu.ppy.sh/beatmapsets/215476#taiko/506018) received the least attention at 16 interactions (10 picks and 6 bans). It's a bit disappointing, but the map has made more than its fair share of appearances in past TWCs; it's certainly not the end of the world as the song would otherwise suggest.

## <a name="matches" id="matches"></a>Tournament Matches

Over the course of last weekend, we saw our fair share of surprise performances from newcomer and veteran teams alike. Since it would be impossible to go through every single match in detail, here are some of the highlights below!

### Saturday

> **The first match on the stream saw the unseeded China squaring off against the top-seeded Australia, resulting in a nail-bitingly close result of 3 — 5!**. osu!mania extraordinaire and Team Captain [Wilben_chan](https://osu.ppy.sh/users/6659363) surprised us all with his skill on the drums, landing many high accuracy scores alongside [7jasmine](https://osu.ppy.sh/users/4112195) to secure an early lead of 2 — 0. Australia was taken by surprise, but remained focused and proceeded to turn the tide as the match went on. A monumental showing from Team China, and they would later prove their strength again by defeating the low seeded Sweden 5 — 3.

Reviewer: Regualr dashes with no spaces.
```suggestion
**The first match on the stream saw the unseeded China squaring off against the top-seeded Australia, resulting in a nail-bitingly close result of 3-5!** osu!mania extraordinaire and Team Captain [Wilben_chan](https://osu.ppy.sh/users/6659363) surprised us all with his skill on the drums, landing many high accuracy scores alongside [7jasmine](https://osu.ppy.sh/users/4112195) to secure an early lead of 2-0. Australia was taken by surprise, but remained focused and proceeded to turn the tide as the match went on. A monumental showing from Team China, and they would later prove their strength again by defeating the low seeded Sweden 5-3.
```

> **Italy vs Brazil was one of the most anticipated matches this weekend, ending in a score of 2 — 5 in favour of Brazil.** Both teams' main roster remained largely unchanged from the previous year, though both sides had improved considerably since. With the score hovering at 2 — 2, the match initially seemed dead even at first. However, it was Brazil's diversity in numbers that gave them the winning edge, which Italy couldn't respond to towards the latter half of the match. Nevertheless, both teams have made it through to the next stage, where they may possibly face each other again?

Reviewer: ```suggestion
**Italy vs Brazil was one of the most anticipated matches this weekend, ending in a score of 2-5 in favour of Brazil.** Both teams' main roster remained largely unchanged from the previous year, though both sides had improved considerably since. With the score hovering at 2-2, the match initially seemed dead even at first. However, it was Brazil's diversity in numbers that gave them the winning edge, which Italy couldn't respond to towards the latter half of the match. Nevertheless, both teams have made it through to the next stage, where they may possibly face each other again?
```

![](/wiki/shared/news/2019-04-26-osu!taiko-World-Cup-Group-Stage-Recap/Italy-Brazil.jpg)

### Sunday

> **Chile and South Korea both went the distance, ending in a tiebreaker win for South Korea!** Having only 2 team members present for a 2v2 match is akin to being cornered against a wall. There is no escape - you can only grit your teeth and play to the best of your ability. Faced with insurmountable odds, Chilean Team Captain [Ulqui](https://osu.ppy.sh/users/1263669) and his stalwart companion [Alepat](https://osu.ppy.sh/users/1164931) fought an uphill battle against the top seeded South Korea. With impressive showings from both sides, it wasn't long before the score reached 4 — 4, and the tiebreaker played. In the end, it was South Korea's mod advantage and superior consistency that secured victory. Fortunately for Chile, they have also made it through to the next stage!

Reviewer: ```suggestion
**Chile and South Korea both went the distance, ending in a tiebreaker win for South Korea!** Having only 2 team members present for a 2v2 match is akin to being cornered against a wall. There is no escape—you can only grit your teeth and play to the best of your ability. Faced with insurmountable odds, Chilean Team Captain [Ulqui](https://osu.ppy.sh/users/1263669) and his stalwart companion [Alepat](https://osu.ppy.sh/users/1164931) fought an uphill battle against the top seeded South Korea. With impressive showings from both sides, it wasn't long before the score reached 4-4, and the tiebreaker played. In the end, it was South Korea's mod advantage and superior consistency that secured victory. Fortunately for Chile, they have also made it through to the next stage!
```

![](/wiki/shared/news/2019-04-26-osu!taiko-World-Cup-Group-Stage-Recap/Chile-SouthKorea.jpg)

> **Just 11 hours later, another tiebreaker match broke out between Singapore and Argentina, with Singapore emerging the victor!**. Both teams showed equal tenacity throughout the match, winning on their own respective picks, eventually taking the score to 4 — 4. The tiebreaker was hard fought, with the score flickering many times throughout. However, it was [Lalamomo4](https://osu.ppy.sh/users/7609510) and [Element118](https://osu.ppy.sh/users/2283992) that emerged victorious, narrowly edging out their Argentinian counterparts with a total score difference of 9,500 points. Many congratulations to Singapore for progressing to the next stage!

Reviewer: ```suggestion
**Just 11 hours later, another tiebreaker match broke out between Singapore and Argentina, with Singapore emerging the victor!** Both teams showed equal tenacity throughout the match, winning on their own respective picks, eventually taking the score to 4-4. The tiebreaker was hard fought, with the score flickering many times throughout. However, it was [Lalamomo4](https://osu.ppy.sh/users/7609510) and [Element118](https://osu.ppy.sh/users/2283992) that emerged victorious, narrowly edging out their Argentinian counterparts with a total score difference of 9,500 points. Many congratulations to Singapore for progressing to the next stage!
```

> **The final match of the day saw Germany vs Canada fighting for the top seed in their group**. With a near-identical roster and a lengthy history of placing on the podium in past years, Team Canada is one of the scariest countries you could ever hope to face in this tournament. Under the fearless leadership of [Ak1o](https://osu.ppy.sh/users/1600041), Germany did well to take the first two points of the match. However, Canada quickly recovered and swung the match to 3 — 4. Backed into a corner, Germany's final pick was risky, and it would prove to be fatal - losing by a score difference of only 4,500 points. It really could have gone the other way. Nevertheless, we will be seeing both teams again next weekend!

Reviewer: ```suggestion
**The final match of the day saw Germany vs Canada fighting for the top seed in their group.** With a near-identical roster and a lengthy history of placing on the podium in past years, Team Canada is one of the scariest countries you could ever hope to face in this tournament. Under the fearless leadership of [Ak1o](https://osu.ppy.sh/users/1600041), Germany did well to take the first two points of the match. However, Canada quickly recovered and swung the match to 3-4. Backed into a corner, Germany's final pick was risky, and it would prove to be fatal—losing by a score difference of only 4,500 points. It really could have gone the other way. Nevertheless, we will be seeing both teams again next weekend!
```

![](/wiki/shared/news/2019-04-26-osu!taiko-World-Cup-Group-Stage-Recap/Germany-Canada.jpg)

--------

And that concludes all the highlights last weekend! If you missed out on any of the action, you can always watch the VODs on the [osu!live Twitch channel](https://www.twitch.tv/osulive/videos).

Let's wave our goodbyes to the 16 teams that were narrowly defeated last weekend. We thank you for participating and hope to see you again next year. Good luck to all the remaining teams in this tournament!

## <a name="ahead" id="ahead"></a>Looking Ahead: Round of 16

> **Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double-elimination tournament, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages.

Reviewer: ```suggestion
**Next week we'll be treated to a total of 8 exciting matches, and you can find the schedule on the [TWC 2019 wiki page](https://osu.ppy.sh/help/wiki/Tournaments/TWC/2019#match-schedule:-round-of-16).** As it is a double-elimination tournament, the Round of 16 is unique in the sense that no team is at risk of being eliminated, though their performance will determine how the remainder of the matches play out in future stages.
```
